include/vkgcDefs.h

```diff
@@ -378,6 +379,14 @@ enum class ThreadGroupSwizzleMode : unsigned {
   Count
 };
 
+/// Represents mapping layout of the resources used in shaders
+enum class ResourceMappingLayout : unsigned {
+  // Compact scheme make full use of all the user data registers.
```

Comment:
nit: can you put these comments on the right of values, just like in the lgc enum?

Reply:
Yes, I will change it . Thanks.